TableSchemer/AccordionScheme.swift

```diff
@@ -8,40 +8,40 @@
 
 import UIKit
 
-public class AccordionScheme<CollapsedCellType: UITableViewCell, ExpandedCellType: UITableViewCell>: BasicScheme<CollapsedCellType> {
+open class AccordionScheme<CollapsedCellType: UITableViewCell, ExpandedCellType: UITableViewCell>: BasicScheme<CollapsedCellType> {
     
-    public typealias AccordionConfigurationHandler = (cell: ExpandedCellType, index: Int) -> Void
-    public typealias AccordionSelectionHandler = (cell: ExpandedCellType, scheme: AccordionScheme, selectedIndex: Int) -> Void
+    public typealias AccordionConfigurationHandler = (_ cell: ExpandedCellType, _ index: Int) -> Void
```

Comment:
Is the `_` here part of adopting S3 api design guidelines? Previously we had the user put in `cell` and `index`.